addons/hide-flyout/userscript.js

```diff
@@ -9,6 +9,7 @@ export default async function ({ addon, console, msg }) {
   let flyoutLock = false;
   let closeOnMouseUp = false;
   let scrollAnimation = true;
+  let notClosedByVisibleDropdownOrInput = true;
```

Comment:
Could this be reversed? Not-variables get confusing quickly especially if they aren't constants. Also visible doesn't need to be in the variable's name.
```suggestion
  let closedByDropdownOrInput = false;
```
Remember to also reverse the values elsewhere too.

Reply:
I was thinking to remove the "not" from the variable name, but decided otherwise. My next commit removes the global variable. After looking closer, the variable need not to be used at all, but instead, it can be substituted for another value.